lita_config.rb

```diff
@@ -58,6 +58,8 @@
     /^!welcome/i => <<MSG,
 Welcome to :256:! Join the conversation. There are many tech-specific channels. (e.g. #code, #testing, #jobs). A lot of silliness goes down in #random.
 
+If you haven't done so already, please upload an avatar and fill out your proflie. We're a friendly group–we don't bite, promise!–but we are a community that likes to know or neighbors!
```

Comment:
I believe that should be 

`but we are a community that likes to know our neighbors!`

You left off the u on the word our.